run_profiles.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+inspec exec inspec-gke-cis-gcp -t gcp:// --input-file inputs.yml &
```

Comment:
If we're running processes in the bg then those processes should output a json for later review.

Reply:
agreed